### PR TITLE
build(package): add ‘utils’ and ‘utilities’ keywords

### LINKED ISSUE #555: Add `utilities` keyword in package.json

### PR DESCRIPTION
**Pull request by Undistraction**
After experimenting with the npms search, I think both utils and utilities are worth adding.

Closes #555

### CODE REVIEW

**Comment by guillaumearm** [comment]
@char0n, should edit github keywords too.

**Comment by Undistraction** [comment]
@guillaumearm Good idea. This can only be done by @char0n

File: package.json [hunk 1]
```diff
@@ -25,7 +25,7 @@
   ],
   "name": "ramda-adjunct",
   "description": "Community-maintained extension of Ramda",
-  "keywords": ["ramda","extensions","addons","cookbook","adjunct","recipe","extras"],
+  "keywords": ["ramda", "utils", "utilities","extensions","addons","cookbook","adjunct","recipe","extras"],
```
Comment (char0n) [inline package.json]: Incosistent spacing between items of the list, some have space some don't

**Comment by Undistraction** [comment]
Prettier wasn't formatting `package.json` on save. This is now formatted correctly and I've added a couple more keywords.

**Comment by Undistraction** [comment]
@char0n Can you approve so I can merge this? Might be worth doing a patch release to get this up on npm. At the moment we aren't even top 10 for a search for 'ramda utils'.

**Pull request merged**